code/modules/antagonists/changeling/changeling.dm

```diff
@@ -692,3 +692,22 @@
 
 	return parts.Join("<br>")
 
+// Changelings spawned from non-changeling headslugs (IE, due to being transformed into a headslug as a non-ling). Weaker than a normal changeling.
+/datum/antagonist/changeling/headslug
```

Comment:
You'll also want to set the soft_antag variable on this antag datum to false if they aren't a true antagonist so that they don't count as an antag towards dynamic, as they'd then be able to alter the probability of midround injections.